apps/src/clientApi.js

```diff
@@ -142,11 +161,21 @@ ApiRequestHelper.prototype.delete = function (localUrl, callback) {
 };
 
 /**
- * @param {!string} localUrl
+ * @param {!string} id
+ * @param {!Object} value
+ * @param {!NodeStyleCallback} callback
+ */
+ClientApiRequest.prototype.update = function (id, value, callback) {
+  this.post('/' + id, value, callback);
+};
+
+/**
+ * @param {!string|number} id Id to delete
```

Comment:
Ahh yes, for some reason I was thinking ! meant optional at the time.

I wouldn't think we would ever actually call this without an id, but the way the code existed before it was possible to do so (on the helper class). I wonder if the better fix would be to have it be required, and remove the undefined check.